docs/windows.md

```diff
@@ -70,26 +70,45 @@ choco install visualstudio2022-workload-vctools
 
 #### Install the Latest Python
 
-Download and install the latest version of Python:
+In Windows, The full installer contains all the Python components and is the
+best option for developers using Python for any kind of project.
+
+For more information on how to use the official installer, please see the
+[full installer instructions](https://docs.python.org/3/using/windows.html#windows-full).
+The default installation options should be fine for use with Gaphor.
+
+1. Install the latest Python version using the
+[official installer](https://www.python.org/downloads/windows/).
```

Comment:
Does this imply that installing Python via Choco or the Windows Store does not work?

Reply:
It should work. I spent a lot of time messing with choco version and pyenv-win. I came to the conclusions that the Python official version is the best for Windows, especially if you need to have multiple versions installed. So I thought I would update our instructions to use this best practice.